src/components/WriterDashBoardTabs.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { useState, useEffect } from "react";
+import useSessionStorage from "../hooks/useSessionStorage";
+
+function WriterDashBoardTabs() {
+  const [isAdmin, setIsAdmin] = useState(false);
+  const [groups] = useSessionStorage("userGroups");
+  const { VITE_ADMIN_GROUP_ID: ADMIN_GROUP_ID } = process;
+  const [currentTab, setCurrentTab] = useState("Unassigned");
+  const handleTabEvents = (tab) => {
```

Comment:
Keep it Simple.

 const handleTabEvents = (tab) => {
      setCurrentTab(tab.target.innerText);
  };

Reply:
addressed